src/test/java/org/openmrs/contrib/qaframework/automation/AllergiesSteps.java

```diff
@@ -59,6 +59,7 @@ public void systemAddsNoKnownAllergies() {
 	@And("a user clicks Remove No Known Allergy icon")
 	public void removeNoKnownAllergy() {
 		allergyPage.removeNoKnownAllergy();
+		allergyPage.waitForPage();
```

Comment:
Same here

Reply:
adding/removing the allergy some times takes longer and often fails via Bamboo. Invoking waitForPage() gives a room for the operations to be successful before the test performs the next action. Have been observing Bamboo builds failures concerning these operations and we would need all the flaws within these tests fixed so that we the module is stable.